x-pack/plugins/elastic_assistant/server/routes/knowledge_base/entries/bulk_actions_route.ts

```diff
@@ -241,9 +294,10 @@ export const bulkActionKnowledgeBaseEntriesRoute = (router: ElasticAssistantPlug
 
           return buildBulkResponse(response, {
             // @ts-ignore-next-line TS2322
-            updated: docsUpdated,
+            updated: transformESToKnowledgeBase(docsUpdated),
```

Comment:
The schema expectation is to return `updated` as an array of `KnowledgeBaseEntryResponse`, so we need to do this transformation.